tools/.golangci.yaml

```diff
@@ -115,7 +115,9 @@ linters-settings: # please keep this alphabetized
       - error-nil
       - expected-actual
       - float-compare
-      - formatter
       - go-require
       - require-error
     enable-all: true
+    formatter:
+      # Require f-assertions (e.g. assert.Equalf) if a format string is used.
```

Comment:
It could be just a nitpick on the comment, but after reading
* this conversation https://github.com/etcd-io/etcd/pull/18741#discussion_r1802658930
* and the references https://github.com/Antonboom/testifylint?tab=readme-ov-file#formatter, https://github.com/Antonboom/testifylint?tab=readme-ov-file#historical-reference.

I think this comment's "format string" part is inaccurate. The suggestion is to use the `*f` functions regardless of whether the message string has formatting or not (if I understood correctly).

Reply:
You're right, even in the documentation from golangci-lint it is not very accurate. I think it something like
> # Require f-assertions (e.g. assert.Equalf) if a message is passed to the assertion.
might be better.

Reply:
How about 


> Require f-assertions (e.g. `assert.Equalf`) if a message is passed to the assertion, even there is no variable-length variables, i.e. require `require.NoErrorf` for both cases below,
> `require.NoErrorf(t, err, "whatever message")`
> `require.NoErrorf(t, err, "whatever message: %v", v)`